libs/geometry/tests/test_tangent_space_mesh.cpp

```diff
@@ -347,6 +347,11 @@ TEST_F(TangentSpaceMeshTest, Lengyel) {
     size_t const vertexCount = mesh->getVertexCount();
     std::vector<quatf> quats(vertexCount);
     mesh->getQuats(quats.data());
+
+    ASSERT_EQ(mesh->getTriangleCount(), CUBE_TRIANGLES.size());
+    std::vector<ushort3> triangles(mesh->getTriangleCount());
+    mesh->getTriangles(triangles.data());
```

Comment:
If the mesh wasn't remeshed, then the caller shouldn't have a reason to call getTriangle, correct? 
Maybe we'd throw an exception in that case actually.

Reply:
actually, I misread the issue.  You are correct!

Reply:
Yup, there's no real reason to call `getTriangle`, it felt wrong to just crash though.